source/configuration/environments/infrastructure/nexus.rst

```diff
@@ -0,0 +1,21 @@
+=====
+Nexus
+=====
+
+Import custom certifications (optional)
+=======================================
+
```

Comment:
Adding a bit of prose explaining the scenario where one would need this might be helpful.

Reply:
e.g. using a firewall inbetween nexus and internet, which is adding their cert to the chain

Reply:
could you please add this into the document? I wasn't asking for myself but kind of as a proxy for future reader of the documentation